pkg/apis/rollouts/validation/validation_test.go

```diff
@@ -284,6 +284,10 @@ func TestValidateRolloutStrategyCanary(t *testing.T) {
 		validRo.Spec.Strategy.Canary.Steps[0].SetWeight = pointer.Int32(10)
 		validRo.Spec.Strategy.Canary.CanaryService = ""
 		validRo.Spec.Strategy.Canary.StableService = ""
+		validRo.Spec.Strategy.Canary.PingPong = &v1alpha1.PingPongSpec{
```

Comment:
Shouldn't we add a new testcase for istio?

Reply:
added new test instead of piggy backing off one